### PR TITLE
ceph-client: introduce new client role

### PR DESCRIPTION
**Pull request by leseb**
By default, this roles will create a ceph config file and get the admin
key. You can optionnally add other users, keys and pools for your tests.

Closes: #769

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by vasukulkarni** [comment]
thanks @leseb

**Comment by leseb** [comment]
test this please

File: roles/ceph-client/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+galaxy_info:
+  author: Sébastien Han
+  description: Installs Ceph RESTAPI
```
Comment (mattt416) [inline roles/ceph-client/meta/main.yml]: `description` needs updating

**Comment by mattt416** [comment]
Cool!  We have similar functionality in openstack-ansible so hopefully we can replace it with this.

**Comment by leseb** [comment]
@mattt416 thanks for the review!

**Pull request merged**